apps/docs/src/app/docs/architecture/page.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Architecture
+nextjs:
+  metadata:
+    title: Architecture
+    description: Overview of the Blobscan's architecture
+---
+
+Blobscan is composed of different components that talk to each other over HTTP through REST endpoints and RPCs.
+
+## Diagram
+
+{% figure  src="/architecture.svg" appendCurrentTheme=true /%}
+
+## Overview
```

Comment:
I think the overview should be before the diagram, so that you have more context when reading the diagram

Reply:
Makes sense